src/main/java/uk/gov/hmcts/probate/services/invitation/controllers/InvitationController.java

```diff
@@ -45,6 +52,18 @@ public String invite(@Valid @RequestBody Invitation invitation,
         return linkId;
     }
 
+    private Invitation decodeURL(Invitation invitation) throws UnsupportedEncodingException {
+        invitation.setExecutorName(decodeURLParam(invitation.getExecutorName()));
+        invitation.setFirstName(decodeURLParam(invitation.getFirstName()));
+        invitation.setLastName(decodeURLParam(invitation.getLastName()));
```

Comment:
Hi Megan, can you move this to the Invitation service and rename decodeURL to something like decodeInvitation, can you also move decodeURLParam, thanks.

Reply:
Done :)